applications/registration-react/src/pages/catalogs-page/catalogs/catalog-item/catalog-item.component.jsx

```diff
@@ -6,15 +6,40 @@ import localization from '../../../../lib/localization';
 import './catalog-item.component.scss';
 
 export const CatalogItem = props => {
-  const { publisherId, type, itemsCount } = props;
+  const { type, itemsCount, linkUri } = props;
+  const betaClassIcon = type === 'concepts' ? 'beta' : '';
+
   const iconClass = cx('catalog-icon', {
     'catalog-icon--dataset': type === 'datasets',
-    'catalog-icon--api': type === 'apis'
+    'catalog-icon--api': type === 'apis',
+    'catalog-icon--concepts': type === 'concepts'
   });
+
+  if (type === 'concepts') {
```

Comment:
det ser ut nesten som helt ant komponent? Er det vits å ha en komponent til alle tre med svært IF, eller kanskje bedre å ha individuelle komponentene? Jeg har ikke svar :) Bare tenk litt

Reply:
Snakket med Øyvind om dette, vi gikk for IF så lenge :)